ROADMAP.md

```diff
@@ -1,43 +1,14 @@
 # Katib 2020 Roadmap
 
-This document provides a high level view of where Katib will grow in 2020. 
+## New Features
 
-The original Katib design document can be found [here](https://docs.google.com/document/d/1ZEKhou4z1utFTOgjzhSsnvysJFNEJmygllgDCBnYvm8/edit#heading=h.7fzqir88ovr).
+- Support Early Stopping [#692](https://github.com/kubeflow/katib/issues/692)
+- Support Advanced NAS Algorithms like DARTs, ProxylessNAS [#461](https://github.com/kubeflow/katib/issues/461)
+- Support Auto Model Compression [#460](https://github.com/kubeflow/katib/issues/460)
+- Support Auto Feature Engineering [#475](https://github.com/kubeflow/katib/issues/475)
 
-# Katib 1.0 Readiness
+## Enhancements
 
-* Stabilize APIs for Experiments
-	* Reconsider the design of Trial Template [#906](https://github.com/kubeflow/katib/issues/906)
-	* Early Stopping [#692](https://github.com/kubeflow/katib/issues/692)
-	* Resuming Experiment [#1061](https://github.com/kubeflow/katib/issues/1061), [#1062](https://github.com/kubeflow/katib/issues/1062)
-* Fully integrate Katib with existing E2E examples:
-	* Xgboost
-	* Mnist
-	* GitHub issue summarization
-* Publish API documentation, best practices, tutorials
-* [Issues list](https://github.com/kubeflow/katib/issues)
-
-# Enhance HP Tuning Experience
-
-The objectives here are organized around the three stages defined in the CUJ:
-
-## 1. Defining Model and Parameters
-
-Integration with KF distributed training components
-* TFJob
-* PyTorch
-* Allow Katib to support other operator types generically [#341](https://github.com/kubeflow/katib/issues/341)
-
-## 2. Configuring a Experiment
-* Supporting additional suggestion algorithms [#15](https://github.com/kubeflow/katib/issues/15)
-
-## 3. Tracking Model Performance
-* UI enhancements: allowing data scientists to visualize results easier
-* Support for persistent model and metadata storage
-	* Ideally users should be able to export and reuse trained models from a common storage
-
-# Test and Release Infrastructure
-
-* Improve e2e test coverage
-* Improve test harness
-* Enhance release process; adding automation (see https://bit.ly/2F7o4gM) 
+- Delete Suggestion deployment after Experiment is finished [#1061](https://github.com/kubeflow/katib/issues/1061)
+- Save Suggestion state after deployment is deleted [#1062](https://github.com/kubeflow/katib/issues/1062)
+- Reconsider the design of Trial Template [#906](https://github.com/kubeflow/katib/issues/906)
```

Comment:
Maybe also add: Change design of NAS Model Constructor? 

As you know, we have model constructor inside Training Container. I think it is not very useful for the user.

If we can construct model somewhere else (Suggestion or Init Container) and send, for example, json representation of the model directly to the training container, user doesn't need to include this part to the training container.

I will create an issue about it soon.

Reply:
LGTM. Looking forward to your issue.